app/assets/javascripts/discourse/app/components/d-select.gjs

```diff
@@ -46,20 +46,26 @@ export default class DSelect extends Component {
     return this.args.value && this.args.value !== NO_VALUE_OPTION;
   }
 
+  get includeNone() {
+    return this.args.includeNone !== false;
```

Comment:
Unless it's strictly `false`. It is still included if `undefined`.

Reply:
Generally the pattern is:

```
get includeNone() {
  return this.args.includeNone ?? false; // or true depending on which default you want, so maybe true here
}
```